pkg/alicloud/client/client.go

```diff
@@ -355,3 +358,78 @@ func (c *vpcClient) DescribeNatGateways(req *vpc.DescribeNatGatewaysRequest) (*v
 func (c *vpcClient) DescribeEipAddresses(req *vpc.DescribeEipAddressesRequest) (*vpc.DescribeEipAddressesResponse, error) {
 	return c.client.DescribeEipAddresses(req)
 }
+
+// vpcClient defines the struct of VPC client
```

Comment:
Change `vpcClient` to `ramClient` or just remove the comment.

Reply:
done